packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +166,18 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		workflowRunning: {
+			handler(bool) {
+				if (bool && this.nodeType!.webhooks) {
```

Comment:
this does not work well.. try `webhookId` (and maybe it's a trigger)
<img width="598" alt="Screen Shot 2021-11-12 at 12 29 19 PM" src="https://user-images.githubusercontent.com/4711238/141460389-28da229f-a06a-4355-be2f-8cc6aca28473.png">

please test more nodes

Reply:
@mutdmour I fixed everything expect this case that you mentioned above. The problem is that some of the nodes has webhookId prop even if they are not triggered nodes. 

I am trying also to use global executionWaitingForWebhook prop as well, but for example, if we have Wait node (which is not trigger node) and Webhook Node (which is actually triggered node), the executionWaitingForWebhook is set to true, but the length of array that measure how many triggered nodes are in the WF, says 2 which is not correct. 
I am going to continue to find proper solution for this. Once I found, I will update the PR as well. 

Btw, thanks for all the directions mentioned above, it helps me a lot.

Reply:
Update: I found solution for it. I am getting all nodes types and check if the nodes belongs to group 'trigger'. I tested with more nodes, and seems like that it is working as expected. :)

Reply:
nice thanks for the fixes. :)